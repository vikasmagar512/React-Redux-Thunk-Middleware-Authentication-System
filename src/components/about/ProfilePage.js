import React,{Component} from 'react';
import createReactClass from  'create-react-class';

class ProfilePage extends Component {
  render() {
    return (
      <div>
        <p>Profile</p>
      </div>
    );
  }
}

export default ProfilePage;