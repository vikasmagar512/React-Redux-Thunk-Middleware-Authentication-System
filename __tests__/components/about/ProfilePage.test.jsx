
import React,{Component} from 'react';
import createReactClass from  'create-react-class';
import PropTypes from 'prop-types'
import { Link, IndexLink } from 'react-router';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import ProfilePage from '../../../src/components/about/ProfilePage';


describe('ProfilePage', () => {
  it('should match its empty snapshot', () => {
    const tree = renderer.create(
      <ProfilePage/>
     ).toJSON();
    expect(tree).toMatchSnapshot();
  });
/*    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<App/>, div);
    });
*/   
}); 