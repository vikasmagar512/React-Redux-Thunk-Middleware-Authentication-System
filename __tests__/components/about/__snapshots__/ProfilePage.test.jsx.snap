// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfilePage should match its empty snapshot 1`] = `
<div>
  <p>
    Profile
  </p>
</div>
`;
